pkg/commands/git.go

```diff
@@ -261,8 +261,13 @@ func (c *GitCommand) RenameCommit(name string) error {
 }
 
 // Fetch fetch git repo
-func (c *GitCommand) Fetch() error {
-	return c.OSCommand.RunCommand("git fetch")
+func (c *GitCommand) Fetch(unamePassQuestion func(string) string, canSskForCredentials bool) error {
```

Comment:
typo: I believe this should be `canAskForCredentials` rather than `canSskForCredentials`